workflow/rules/common.smk

```diff
@@ -1,4 +1,14 @@
-def bids (root=None, prefix=None, suffix=None, subject=None, session=None,include_subject_dir=True,include_session_dir=True,template=None,**entities):
+def bids(
```

Comment:
was only using this bids function defined in common.smk because snakebids wasn't around back then.. Probably better off just using snakebids unless there is something custom here?

Reply:
Nothing custom, other than adding in `hemi` as an entity. Was thinking the same thing of replacing this with `snakebids`, but wanted to hold off until got Roy's funcparc part also implemented.